### PR TITLE
Upgrade builder to cosmwasm/optimizer:0.15.0

### PR DESCRIPTION
**Pull request by webmaster128**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -4,12 +4,6 @@ version = "0.1.0"
 authors = ["{{authors}}"]
 edition = "2021"
 
-exclude = [
-  # Those files are rust-optimizer artifacts. You might want to commit them for convenience but they should not be part of the source code publication.
-  "contract.wasm",
-  "hash.txt",
-]
```
Comment (webmaster128) [inline Cargo.toml:11]: Those files are not created anymore since rust-optimizer 0.10.0 from 3 years ago

**Review by chipshort — APPROVED** [review]
LGTM

File: Cargo.toml (hunk 1 above)
Comment (chipshort) [reply Cargo.toml:11]: What a coincidence, I was wondering about that this morning before seeing this PR.

Comment (webmaster128) [reply Cargo.toml:11]: Relicts from an ancient history before you joined :)

**Comment by memilis007** [comment]
Hi

**Comment by memilis007** [comment]
Good

**Pull request merged**